### DISCUSSION TITLE
Tiddlyclip rule to clip local mp4 files using relative path for annotation by mediaplayer plug in

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#113] [Q&A] [answered]
I want to create a Tiddlyclip rule to clip local mp4 files using relative path for annotation by mediaplayer plug in.
This is what I tried, but its not working.

<img width="499" alt="image" src="https://user-images.githubusercontent.com/67494083/217356586-42299470-6527-4731-8236-567cba81c7ef.png">

<img width="555" alt="image" src="https://user-images.githubusercontent.com/67494083/217356638-7f6f339a-f6dd-407c-b2e3-9ebe8b089feb.png">

<img width="559" alt="image" src="https://user-images.githubusercontent.com/67494083/217356677-b98d8aa2-994a-477b-a390-45a748069ca8.png">

<img width="831" alt="image" src="https://user-images.githubusercontent.com/67494083/217356710-a0ffbe64-74a6-4836-ac9f-e02a0db56650.png">

<img width="1368" alt="image" src="https://user-images.githubusercontent.com/67494083/217356779-86d1c389-0c98-40f9-ab35-08c914ca8203.png">

[Here](https://tiddlyclip-discussions.tiddlyhost.com/) is a demo

**Comment by buggyj** [comment]
I think you need

`$location=((*@sublink(@linkURL)*))`

**Reply by arunnbabu81** [reply]
> I think you need
> 
> `$location=((*@sublink(@linkURL)*))`

When I am using this, I am getting like this

<img width="1386" alt="image" src="https://user-images.githubusercontent.com/67494083/217536624-60895b67-327d-4ae1-b072-4879d361ca51.png">

How to avoid the `%20` in the filename

**Comment by buggyj** [comment, marked as answer]
```
#rule2=/%20/g
//note there is a space below
#term2= 
$location=((*$location/#rule2/#term2*))
```
